handlers/deletePlayer.go

```diff
@@ -11,6 +12,10 @@ type DeletePlayerRequest struct {
 	GameId string `json:"game_id"`
 }
 
+type DeletePlayerResponse struct {
+	Game truco.Game `json:"game"`
+}
+
```

Comment:
Haven't thought through how the frontend will use this one yet, but no harm returning `Game` ✅